bin/parse_viral_pred.py

```diff
@@ -277,36 +263,38 @@ def parse_virus_sorter2(sorter_files, vs_cutoff):
         how="left",
         suffixes=("_score", "_boundary"),
     ).dropna(subset=["shape"])
-        
+
     with open(final_combined_fa_file) as handle:
         for record in SeqIO.parse(handle, "fasta"):
             clean_name = record.id.split("|")[0]
-            max_score = meta["max_score"].get(clean_name, None)
-            circular = meta["shape"].get(clean_name, None)
+            record_name = record.id  # does include ||
+            max_score = meta["max_score"].get(record_name, None)
+            circular = meta["shape"].get(record_name, None)
             prange = [
-                meta["trim_bp_start"].get(clean_name, None),
-                meta["trim_bp_end"].get(clean_name, None),
+                meta["trim_bp_start"].get(record_name, None),
+                meta["trim_bp_end"].get(record_name, None),
             ]
+
             if not circular or not max_score:
                 continue
             circular = circular.startswith("circular")
-
+            
             if "partial" in record.id:
-                # add the prophage position within the contig
                 record.id = clean_name
+                # add the prophage position within the contig
                 prophages.setdefault(clean_name, []).append(
                     Record(record, "prophage", circular, prange)
                 )
             else:
                 record.id = clean_name
                 if float(max_score) >= float(vs_cutoff):
-                    high_confidence[record.id] = Record(record, "high_confidence", circular)
+                    high_confidence[clean_name] = Record(record, "high_confidence", circular)
                 else:
-                    low_confidence[record.id] = Record(record, "low_confidence", circular)
-
-    print(f"Virus Sorter found {len(high_confidence)} high confidence contigs.")
-    print(f"Virus Sorter found {len(low_confidence)} low confidence contigs.")
-    print(f"Virus Sorter found {len(prophages)} putative prophages contigs.")
+                    low_confidence[clean_name] = Record(record, "low_confidence", circular)
+    number_of_phages = sum([len(prophages[i]) for i in prophages])
+    print(f"Virus Sorter2 found {len(high_confidence)} high confidence contigs.")
+    print(f"Virus Sorter2 found {len(low_confidence)} low confidence contigs.")
+    print(f"Virus Sorter2 found {number_of_phages} putative prophages contigs.")
```

Comment:
What is going to be printed if input is virsorter 1?

Reply:
Ignore this, I just went throuth the whole script and realised there are two separated functions, one for each VS version